docs/configuring-kubelinter.md

```diff
@@ -62,3 +62,59 @@ example,
 > [!TIP]
 > `exclude` always takes precedence, if you include and exclude the same check,
 > KubeLinter always skips the check.
+
+## Run custom checks
+
+You can write custom checks that are based on existing templates.
```

Comment:
Can we also insert a link to all the existing templates here? And also please mention that params definitions are also taken from the respective templates.